facebook/OpenBIC
# op2: op: Enable config of GPIO about recovery retimer

# Description
- Enable the GPIO settings related to the recovery retimer.

# Motivation
- To recovery retimer, EE add the new GPIO.

# Test plan
- Build code: Pass
- Check GPIO setting: Pass

# Log
1. Get GPIO setting.
- Before enable uart:~$ platform gpio get 100
gpio[100] get failed!
uart:~$ platform gpio get 101
gpio[101] get failed!
uart:~$ platform gpio list_group GPIO0_M_P
...
[100][GPIO0_M_P 4  ] gpio_disable                       : -- | i/o       | NA
[101][GPIO0_M_P 5  ] gpio_disable                       : -- | i/o       | NA
...

- After enable uart:~$ platform gpio get 100
[100] SMB_LS_MUX_EN                      : PP  | output(O) | 0(0)
uart:~$ platform gpio get 101
[101] SELECT_SMB_MUX_N                   : PP  | output(O) | 0(0)
uart:~$ platform gpio list_group GPIO0_M_P
...
[100][GPIO0_M_P 4  ] SMB_LS_MUX_EN                      : PP | output(O) | 0(0)
[101][GPIO0_M_P 5  ] SELECT_SMB_MUX_N                   : PP | output(O) | 0(0)
...
